Fix "implicitly declaring library function 'tolower' with type 'int (int)'"
--- src/xwin.c.orig	2021-01-25 06:13:51.000000000 -0600
+++ src/xwin.c	2021-01-25 11:18:27.000000000 -0600
@@ -45,6 +45,7 @@
 #include <stdint.h>
 #include <limits.h>
 #include <sys/stat.h>
+#include <ctype.h>
 sigset_t block_alarm;
 sigset_t waiting_mask;
 sigset_t base_mask;
